### PR TITLE
DROOLS-2140: [Guided Decision Table] Dragging table should be disabled when pinned

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2140

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko FYI.. it'd be nice if I you was _in_ AppFormer so I could add you too!

**Comment by manstis** [comment]
@ederign Could you please add @jomarko to the AF _team_ in GitHub?

**Comment by jomarko** [comment]
@manstis @ederign you have my bless for adding into this group :) anyway, looking on the PR, thanks.

File: uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/widget/dnd/GridWidgetDnDMouseMoveHandlerTest.java [hunk 1]
```diff
@@ -281,13 +292,42 @@ public void findMovableGridWhenNoColumnOrRowOperationIsDetectedAndGridIsPinned()
...
+                                                verify(state).setActiveGridWidget(eq(gridWidget));
+                                                verify(state).setOperation(eq(GridWidgetHandlersOperation.GRID_MOVE_PENDING));
+                                                assertEquals(gridWidget,
+                                                             state.getActiveGridWidget());
+                                                assertEquals(GridWidgetHandlersOperation.GRID_MOVE_PENDING,
+                                                             state.getOperation());
+                                            });
+    }
+
+    public void doFindMovableGridWhenOverDragHandle(final boolean isPinned,
```
Comment (jomarko) [inline uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/widget/dnd/GridWidgetDnDMouseMoveHandlerTest.java]: Please, use private access.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by manstis** [comment]
@jomarko Access modifier corrected in subsequent commit.. Damn the copy and paste gods!

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
Jenkins please execute full downstream build.

**Comment by manstis** [comment]
Jenkins please execute full downstream build.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by manstis** [comment]
Jenkins please execute full downstream build.

**Comment by manstis** [comment]
Jenkins please execute full downstream build.

**Comment by manstis** [comment]
@cristianonicolai Hi.. jbpm seems broken preventing downstream builds.

https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/uberfire-downstream-pullrequests/130/console
```
 [ERROR] Errors in 'org/jbpm/workbench/pr/client/editors/instance/signal/ProcessInstanceSignalPresenter.java'
14:53:42 [INFO]       [ERROR] Line 48: Failed to resolve 'org.jbpm.workbench.pr.client.resources.i18n.Constants' via deferred binding
```
and other similar messages.

**Comment by cristianonicolai** [comment]
Jenkins please execute full downstream build.

**Comment by cristianonicolai** [comment]
@manstis thanks, fixing it in here: https://github.com/kiegroup/jbpm-wb/pull/945

**Comment by cristianonicolai** [comment]
Jenkins please execute full downstream build.

**Comment by ederign** [comment]
@manstis sorry for the delay to merge this.

**Pull request merged**